engine/src/main/java/org/terasology/rendering/dag/DownSampleSceneAndUpdateExposureNode.java

```diff
@@ -95,18 +99,16 @@ public void initialise() {
     // TODO: verify if this can be achieved entirely in the GPU, during tone mapping perhaps?
     @Override
     public void process() {
-        sceneOpaque = frameBuffersManager.getFBO("sceneOpaque");
-        scenePrePost = frameBuffersManager.getFBO("scenePrePost");
         if (renderingConfig.isEyeAdaptation()) {
-            PerformanceMonitor.startActivity("rendering/downsamplesceneandupdateexposure");
+            PerformanceMonitor.startActivity("rendering/updateExposure");
 
-            downsampleSceneInto1x1pixelsBuffer();
+            downSampleSceneInto1x1pixelsBuffer();
```

Comment:
Hehe, actually in this case it would be correct to keep it as "downsample" as it is actually a one-word verb, like upscaling for example. But don't bother, it's ok to leave it as it is.